docker/Dockerfile

```diff
@@ -70,10 +70,9 @@ RUN /bin/sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc)
     ros-${DIST}-velodyne-simulator \
     ros-${DIST}-rqt \
     ros-${DIST}-rqt-common-plugins \
- && rosdep init \
  && apt clean
 
-RUN rosdep update
+RUN /bin/bash -c ". /opt/ros/${DIST}/setup.bash"
```

Comment:
I noticed rosdep is no longer installed by default so `rosdep init` and `rosdep update` throw errors. The [ROS Melodic Installation Instructions](http://wiki.ros.org/melodic/Installation/Ubuntu) resolve this by installing `python-rosdep` explicitly. I'm assuming this edit indicates we don't really need to do this--would you mind explaining why?

Reply:
@caguero Just bumping this. I'm confused because it looks like running setup.bash replaces the calls to rosdep, but setup.bash seems just to set environment variables, and since it is on it own line I expect these to disappear before they are used. Can you say what's going on here?

Reply:
Oh, if I remember correctly, I saw this error:

```
/bin/sh: 1: rosdep: not found
```

And I removed these two lines as a quick test but I forgot to restore it. I think the proper way to do it is, as you suggested, to install `python-rosdep`.

Reply:
It should be restored now. By the way, I had to change the way Docker is installed. The old way doesn't work anymore (maybe something changed in Ubuntu) and now I'm using another Github action to do it.

Reply:
Thanks! Good to know about the new Docker install commands.

I think line 79 can be eliminated. It appears to be setting environment variables, and these don't persist between RUN commands in Docker files. I ran a quick test by dropping it from my local file and the image still appears to build fine and runs vrx.launch without error.

Reply:
Thanks! Removed in d356499.